griptape/tasks/base_rule_task.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Any
+
+from attr import define, field
+
+from griptape.rules import Ruleset, Rule
+from griptape.tasks import BaseTask
+
+
+@define
+class BaseRuleTask(BaseTask, ABC):
```

Comment:
The content of BaseRuleTask is copied directly from BaseTextInputTask, no new functionality here.